source/_integrations/fing.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: Fing
+description: "Instructions on integrating your Fing Agent with Home Assistant"
+ha_release: 2024.9.1
+ha_category: 
+    - Presence detection
+ha_platforms:
+    - device_tracker
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@Lorenzo-Gasparini'
+ha_domain: fing
+ha_integration_type: integration
+---
+
+The [Fing Agent](https://www.fing.com/fing-agent/) integration enables you to retrieve device data from your network, create your personal device inventory and use it to optimize your automations.
```

Comment:
```suggestion
The [Fing](https://www.fing.com/) {% term integration %} enables you to retrieve device data from your network, create your personal device inventory, and use it to optimize your automations. The **Fing integration** allows you to retrieve device information through the [local API](https://www.fing.com/developers/local-api/) exposed by Fing agents ([Fing Agent](https://www.fing.com/fing-agent/), Fingbox, Fing Desktop). 
```
If I understand it correctly from the core PR, the integration is called Fing, and not Fing Agent. I rephrased it a bit to clarify the difference between Fing and Fing Agent. Please feel free to rephrase if needed.